code/game/gamemodes/marker/evac_points.dm

```diff
@@ -32,12 +39,37 @@
 		evac_points = evac_threshold
 		//No addtimer call here, so we'll stop gaining points now
 	else
+		check_admin_warnings()
 		pointgain_timer = addtimer(CALLBACK(src, .proc/charge_evac_points), 1 MINUTE, TIMER_STOPPABLE) //Shuttle was unable to be called. Try again recursively.
 	return FALSE
 
-
+//proc that handles messages to admins regarding close evac time
+/datum/game_mode/marker/proc/check_admin_warnings()
+	var/time_till_evac = (evac_threshold - evac_points) * minutes_per_point MINUTES	//for Nanako, this formula here needs checking
```

Comment:
You made a proc for this elsewhere, use that instead of pasting code around

Reply:
Added MINUTES to this proc, removed from places it's called.